src/Color/ColorConverter.php

```diff
@@ -199,8 +213,8 @@ private function processHashColor($color)
 	/**
 	 * @param $mode
 	 * @param mixed[] $cores
-	 *
-	 * @return bool|\float[]
+	 * @return bool|float[]
+	 * @throws \Mpdf\MpdfException
```

Comment:
I discourage `@throws` PHPDoc tag, it is a mess to maintain, better to ommit it.

Reply:
Could it be useful if automatically docs are generated from the source?

Reply:
IMHO not too much. There is not enough difference in exceptions thrown (now there are about 2 or 3 types) and when the exception change in code is not reflected in the phpdoc, the generated doc would be off as well.

Reply:
I agree it's a pity to maintain, reverted the change :)